### ISSUE TITLE
Can you provide any reports or any papers to detail this work?

### ISSUE THREAD

**Issue by deepmeng** [quan-dao/pmbm-filter#1]
Hi, Quan,  thanks for your excellent work. Can you provide any reports or any papers to detail this work? I believe that readers will be very interested in the principles and the adopted methods of this work.

**Comment by quan-dao** [comment]
Hello @deepmeng,
Sorry for the late respond.
The theory of PMBM filter is presented in https://arxiv.org/abs/1802.09975 and https://arxiv.org/pdf/1703.04264.pdf
If you find the theory in these papers are hard to follow you can checkout the authors' course on Edx: https://youtu.be/CsYIlsGLqg4
My implementation of PMBM filter is based on https://github.com/erikbohnsack/pmbm
